### ISSUE TITLE
cloneInstace() doesn't copy store functions

### ISSUE THREAD

**Issue by ronami** [i18next/i18next#859]
Cloning an instance via .cloneInstance and trying to call any of the copied store functions (https://github.com/i18next/i18next/blob/master/src/i18next.js#L100) such as `clonedInstance.getResourceBundle()` would result in an error.

You can still call `clonedInstance.store.getResourceBundle()` but it would be better to go with the api docs.

Is there a reason these functions are not copied to the cloned instance?

I can easily make a PR to support that, please let me know if any of this makes sense and I will.

Thanks!

Edit: I noticed it can be accessed through `i18next.services.resourceStore` (which is also documented), but it would be easier if a cloned instance's api would match the original instance, wouldn't it?

**Comment by jamuhl** [comment]
no...that's an issue...rather sure if we set callback here (https://github.com/i18next/i18next/blob/master/src/i18next.js#L282) to noop if not set it will run through https://github.com/i18next/i18next/blob/master/src/i18next.js#L100 and set that API functions.

if you like you can submit a PR or i will fix this for next version.

**Comment by ronami** [comment]
Here's a PR: https://github.com/i18next/i18next/pull/860.

I wasn't sure how it would be best to test something like that, best I could come up with is something like the following, but it requires duplicating the storeApi keys, along with not really testing that the values are the correct ones. I didn't include those tests in my PR.

```js
      it('it has shortcut store functions', () => {
        const storeApi = ['getResource', 'addResource', 'addResources', 'addResourceBundle', 'removeResourceBundle', 'hasResourceBundle', 'getResourceBundle'];

        storeApi.forEach(fcName => {
          expect(newInstance[fcName]).to.be.a('function');
        });
      });
```

**Comment by jamuhl** [comment]
published in i18next@5.0.0
